docs/examples/browser/README.md

```diff
@@ -0,0 +1,83 @@
+Most certainly, you can the Portablegabi API in the browser.
+But you have to make it compatible with the browser API first, e.g. by bundling it with webpack.
+
+Please note that **bundling your JS code does not suffice unfortunately**.
+You also need to **serve the Portablegabi WASM file** [`main.wasm`] located in `/build/src/wasm`.
+We recommend using our webpack configuration below which copies the WASM to your `dist` directory, in which your `bundle.js` should be stored.
+Please **check out the FAQ** in case your `node_modules` directory is not in the root level of your app, or you have a custom webpack output folder apart from `dist`.
+
+## General Setup
+
+1.  Bundle your JavaScript or TypeScript file that runs all the, e.g. with `yarn webpack --entry <your_script_file> --out <desired_output_file>
```

Comment:
`that runs all the` what? feels like something is missing here

Reply:
Resolved in 1927391